### ISSUE TITLE
Can you please add the dice loss for NER

### ISSUE THREAD

**Issue by albertnanda** [fursovia/self-adj-dice#3]
Shape:
        - logits: `(N, L,C)` where `N` is the batch size, L is the tokens and `C` is the number of classes.
        - targets: `(N,L)` where each value is in [0, C - 1]

**Comment by fursovia** [comment]
@albertnanda I've released `1.0.3` version with the new `reduction` argument. Please try the following:

```python
import torch
from sadice import SelfAdjDiceLoss

criterion = SelfAdjDiceLoss(reduction="none")
# (batch_size, num_tokens, num_classes)
logits = torch.rand(128, 40, 10, requires_grad=True)
targets = torch.randint(0, 10, size=(128, 40))

loss = criterion(logits.view(-1, 10), targets.view(-1))
loss = loss.reshape(-1, 40).mean(-1).mean()
loss.backward()
```
